pkg/explorer/esplora/unspents_test.go

```diff
@@ -44,10 +44,6 @@ func TestGetUnspents(t *testing.T) {
 }
 
 func TestSelectUnspents(t *testing.T) {
```

Comment:
It may be a sleep problem maybe?

Reply:
Not sure how this test is supposed to work, 
if we assume that utxo created after faucet is 1 * math.Pow10(8), how target is targetAmount := uint64(1.5 * math.Pow10(8)) can be bigger? also in assertion we are expecting 2 utxo, maybe I am missing something ...

Also, if i skip this test CI finishes successfully, if not I am getting RACE issue with crawler and this test is passing.
On the other hand locally i dont have issue with crawler RACE condition, only TestSelectUnspents fails with wrong assertions.

Reply:
they are intended to work because `newTestData()` returns always the same address and key once it has created. So TestGetUnspents should send 1 LBTC to the test address, and then the same should do TestSelectUnspents, so the test should pass because the testing address is owning 2 LBTC... but it doesn't look good as it is to me 😅